src/coreclr/src/debug/daccess/dacfn.cpp

```diff
@@ -104,6 +104,8 @@ DacWarning(__in char* format, ...)
     OutputDebugStringA(text);
 }
 
+#pragma warning (push)
+#pragma warning (disable: 4297)
```

Comment:
I wonder why this was not needed before and I wonder if throwing the exception from the functions below will actually work as expected. My worry is that the compiler might exclude EH info for these.

Reply:
I just tested this out again locally by removing this pragma and it didn't emit the warning (I added this when I picked this up again since it had started warning randomly). I'll remove the pragmas so we can make sure to not miss the warning if it pops again.